views/tracks.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <div class="row">
+    {{#each tracks}}
+      <div class="tracksCard">
+        <img src="{{this.images.[0].url}}" alt="{{this.name}}" />
```

Comment:
cool -> I didn't even know that [i] mixed in with . notation was a thing in HBS :) today I learned :+1: